packages/angular/cli/src/commands/mcp/tools/modernize.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { z } from 'zod';
+
+interface Transformation {
+  name: string;
+  description: string;
+  documentationUrl: string;
+  instructions?: string;
+}
+
+const TRANSFORMATIONS: Array<Transformation> = [
```

Comment:
This should be good to test and experiment.  Long term we probably want to try to pull the information from `@angular/core` directly at runtime.  This would avoid needing to keep this information synchronized.